### DISCUSSION TITLE
"html.autoClosingTags": false is not working for volar.

### DISCUSSION THREAD

**Discussion by zjlovezj** [vuejs/language-tools#1090] [General]
I'm not sure whether autoClosingTags is mandatory for volar.

But in VS code set "html.autoClosingTags": false is not working when volar is enabled.

Is it a feature or bug?

**Comment by johnsoncodehk** [comment]
It should be fixed in https://github.com/johnsoncodehk/volar/commit/b32be70880e5ca5765d23d9085896f7af7b29598, thanks for the report!

**Reply by zjlovezj** [reply]
Thank you for this great tool.
